apps/block_scout_web/lib/block_scout_web/templates/tokens/overview/_metatags.html.eex

```diff
@@ -1,5 +1,5 @@
 <title>
-  <%= "#{token_name(@token)} (#{token_symbol(@token)}) - #{LayoutView.subnetwork_title()} - BlockScout" %>
+  <%= "#{Gettext.gettext(BlockScoutWeb.Gettext, token_name(@token))} - #{LayoutView.subnetwork_title()} - BlockScout" %>
```

Comment:
question about `getText` if there is not a translation found does it just output the input unchanged? or will it do something like "missing translation" 

of course here we want the former so tokens like cUSD and cBRL (future thinking) are handled nicely.

Reply:
Just checked and it outputs the original input.  It may fail the "gettext" tests, but I'm not 100% sure...

Reply:
If there is no `msgstr` provided, it should output the value of `msgid`, yes.